lib/pages/settings/model/request.dart

```diff
@@ -1,9 +1,17 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
 class Request {
-  Request(this.userId, this.requestBody);
+  Request(this.userId, this.requestBody, this.timestamp);
 
   /// The user who created this request
   final String userId;
 
   /// The body of the request
   final String requestBody;
+
+  /// Boolean value represented whether the request has been processed by mods
+  final bool processed = false;
```

Comment:
Add this as a (named) constructor parameter. A `final` variable with a set value that you can't even set when instantiating isn't more useful than a simple constant. We'll need to be able to set this when we read requests from the db and convert them to Request instances.